Source/WebCore/page/DOMTimer.cpp

```diff
@@ -93,7 +93,7 @@ class DOMTimerFireState {
     static DOMTimerFireState* current;
 
 private:
-    ScriptExecutionContext& m_context;
+    WeakPtr<ScriptExecutionContext> m_context;
```

Comment:
I guess this one can be RefPtr since DOMTimerFireState is always allocated in stack.